### ISSUE TITLE
Cannot log in using the default credentials

### ISSUE THREAD

**Issue by johann-petrak** [doccano/doccano#704]
Trying to run on Ubuntu 18.04 LTS.
After cloning c31dd82c31f38f0398b2699d3303b4d676201106 and modifying the docker-compose.dev.yml file to contain the following lines instead of the originals:

```
version: "3"
[...]
    ports:
      - 8999:3000
```
I am running 
`sudo docker-compose -f docker-compose.mydev.yml up `
and after a while I can browse to localhost:8999

When I click "SIGN IN" button I get the sign in dialog. When I then enter the credentials admin/password I get the error that they are incorrect and I cannot log in.

**Comment by johann-petrak** [comment]
This worked as expected when using the docker (not docker-compose method).

**Comment by icoxfog417** [comment]
Could you please tell us your environment information following the [ISSUE_TEMPLATE](https://github.com/doccano/doccano/blob/master/.github/ISSUE_TEMPLATE/01-question.md) and you succeeded or not if you do not change the port number.

**Comment by johann-petrak** [comment]
Operating System: Ubuntu 18.04 with current updates
    Python Version Used: I am using Python 3.7 via Anaconda, but I would have thought that when running the docker images, they use whatever is used in there
    When you install doccano: (?? not sure what this means)
    How did you install doccano (Heroku button etc): as I wrote, using `sudo docker-compose -f docker-compose.mydev.yml up ` from the checked out repo with the modified docker-compose.mydev.yml file as described. 

The odd thing is that the login screen I get using "docker-compose" looks very different from the screen I get when using docker.

This is the initial screen on 127.0.0.1:8999 when using docker-compose:
![doccano-docker-compose](https://user-images.githubusercontent.com/619106/79685620-71f7ca00-823a-11ea-81a9-a52b7b0a8b7f.png)
Also 127.0.0.1:8999/admin gives a 404 and is not found.

This is the initial screen on 127.0.0.1:8999 when using the docker image:
![doccano-docker](https://user-images.githubusercontent.com/619106/79685695-ed597b80-823a-11ea-9eaf-1151303eadde.png)
Also http://127.0.0.1:8999/admin provides the Django login page.

**Comment by Hironsan** [comment]
> When I click "SIGN IN" button I get the sign in dialog. When I then enter the credentials admin/password I get the error that they are incorrect and I cannot log in.

This is caused by the CORS policy. When we use `docker-compose.dev.yml`, it mainly run two servers: frontend and backend API. When we submit username and password in the frontend, it is processed in the backend API. In other words, the two servers are communicating. At that time, the backend API restricts cross-origin HTTP requests for security reasons. In the initial settings, the backend allows only port `3000`. That's why you could not login.

If you want to use ports other than `3000`, you need to add `127.0.0.1:8999` to `CORS_ORIGIN_WHITELIST`:

https://github.com/doccano/doccano/blob/c31dd82c31f38f0398b2699d3303b4d676201106/app/app/settings.py#L314-L318

Or, you can use `CORS_ORIGIN_ALLOW_ALL=True` to allow any access.
- [django-cors-headers](https://github.com/adamchainz/django-cors-headers#configuration)

**Comment by Hironsan** [comment]
> The odd thing is that the login screen I get using "docker-compose" looks very different from the screen I get when using docker.

Docker version is older version. I recommend you use `docker-compose` version.

**Comment by johann-petrak** [comment]
I find it odd that there is something in a configuration file that cannot be configured really and where there is also no documentation for how to make it work (except this issue now). 
Maybe actually fixing this could involve adding the information from this issue into the docker-compose yaml files as a comment or a comment that is a pointer to some documentation page or something?

**Comment by Hironsan** [comment]
Sorry for the insufficient documentation.

I think replacing `CORS_ORIGIN_WHITELIST` with `CORS_ORIGIN_ALLOW_ALL` is better approach.

**Comment by johann-petrak** [comment]
I tried this using `CORS_ORIGIN_ALLOW_ALL` and I was now able to log into doccano after running it with docker-compose.

However, I still do not understand how to create more users: within doccano existing users are needed, but the url http://localhost:8999/admin is not found (I used this in the docker version to create new users). It shows a screen with the message "Whoops, 404" and this screen also shows the "SIGN IN" button on the upper right even though I already was logged in in that browser session in a different tab!

I also noticed that interacting with doccano after that 404 screen was shown started to become very slow and unresponsive, not sure if this is related.
